matter/wifi/dhcp_client.cpp

```diff
@@ -98,10 +99,10 @@ uint8_t dhcpclient_poll(void *arg)
         dhcp_state = DHCP_ADDRESS_ASSIGNED;
 
         EFR32_LOG("DHCP IP: %d.%d.%d.%d",
-                  (netif->ip_addr.u_addr.ip4.addr & HEX_VALUE_FF),
-                  ((netif->ip_addr.u_addr.ip4.addr >> SHIFT_8) & HEX_VALUE_FF),
-                  ((netif->ip_addr.u_addr.ip4.addr >> SHIFT_16) & HEX_VALUE_FF),
-                  ((netif->ip_addr.u_addr.ip4.addr >> SHIFT_24) & HEX_VALUE_FF));
+                  (NETIF_IPV4_ADDRESS & HEX_VALUE_FF),
+                  ((NETIF_IPV4_ADDRESS >> SHIFT_8) & HEX_VALUE_FF),
+                  ((NETIF_IPV4_ADDRESS >> SHIFT_16) & HEX_VALUE_FF),
+                  ((NETIF_IPV4_ADDRESS >> SHIFT_24) & HEX_VALUE_FF));
```

Comment:
Revert this change. The modified code is not what i had in mind.

Reply:
The idea proposed by Mathieu was to make a macro or macros that can be reused